__tests__/app.test.js

```diff
@@ -131,4 +131,42 @@ describe('api/articles', () => {
     .get('/api/notarticles')
     .expect(404)
   })
+})
+
+describe('api/articles/:article_id/comments', () => {
+  test('GET 200: returns an array of comments by article ID', () => {
+    return request(app)
+    .get(`/api/articles/9/comments`)
+    .expect(200).then(({body})=> {
+      const commentArray = body.comments
+
+      expect(Array.isArray(commentArray)).toBe(true) 
```

Comment:
technically this check is redundant because body.comments has to be an array in order for the forEach to run, you can remove this assertion